Remove tracing without limits occurrences (#13350)
DataDog/documentation: content/en/account_management/billing/apm_tracing_profiler.md

[APM & Continuous Profiler][1] powers you to find service bottlenecks, analyze distributed traces and code performance across your microservices architecture.

> There are two options available for pricing, depending on whether APM and Profiling are bundled. Control the stream of ingested data with the [ingestion controls][2] parameters and which spans to retain for a longer time period by indexing it with [tagged-based retention filters][3].

Reviewer: ```suggestion
There are two options available for pricing, depending on whether APM and Profiling are bundled. Control the stream of ingested data with [ingestion controls][2]. Control which spans are kept for the retention period by indexing them with [tagged-based retention filters][3].
```

[10]: https://docs.datadoghq.com/account_management/billing/serverless/#serverless-functions
[11]: /account_management/billing/
[12]: /tracing/setup_overview/setup/java/?tab=containers#configure-the-datadog-agent-for-apm

> [13]: /tracing//trace_retention/

Reviewer: ```suggestion
[13]: /tracing/trace_retention/
```
